tests/analyses/test_api.py

```diff
@@ -27,6 +27,13 @@ def files(test_files_path, tmp_path):
     return data
 
 
+@pytest.fixture
+def more_files(test_files_path, tmp_path):
```

Comment:
This does the exact same thing as `files`. Get this down to single fixture. The best solution might be to return a function from the fixture.